framework/src/media/Kconfig

```diff
@@ -8,7 +8,7 @@ config MEDIA
 	default n
 	depends on AUDIO
 	select HAVE_CXX
-	select HAVE_CXXINITIALIZE
+	select HAVE_CXXINITIALIZE if BUILD_FLAT
```

Comment:
How about loadable xip? Isn't it the same as flat?

Reply:
In the case of flat, HAVE_CXXINITIALIZE does static constructors initialization. In the case of loadable static constructors are initialized in binfmt code for both xip loadable and elf loadable and is not dependent on HAVE_CXXINITIALIZE.

Reply:
Oh ok, Loadable XIP does with BINFMT_CONSTRUCTORS as same as loadable ELF.
Then, don't you need a selection for that?
```
select BINFMT_CONSTRUCTORS if (ELF || XIP_ELF)
```

Reply:
https://github.com/Samsung/TizenRT/blob/cf41d8ad8333a95dc375331578c7d5920d4acc9d/lib/libxx/Kconfig#L51-L57

They are selected be default when we enable c++, so we need not add it here.

Reply:
Then, why do we keep this `select HAVE_CXXINITIALIZE if BUILD_FLAT` here?

Reply:
Agreed, removed that line.